cmd/domain/generate.go

```diff
@@ -0,0 +1,150 @@
+package domain
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/bccfilkom-be/bccli/internal/file"
+	"github.com/bccfilkom-be/bccli/internal/template"
+	"github.com/gobeam/stringy"
+	"github.com/spf13/cobra"
+)
+
+type Data struct {
+	Domain       string
+	DomainStruct string
+}
+
+type appOptions struct {
+	handler    bool
+	repository string
+	usecase    bool
+}
+
+var (
+	appOpts  appOptions
+	database string
+)
+
+var generateCmd = &cobra.Command{
+	Use:   "generate",
+	Short: "Generate domain components like handler, usecase, and repository.",
+	Long: `Generate domain components like handler, usecase, and repository.
+You can choose to create specific components or generate all at once.`,
```

Comment:
what it means by or? which one's the default

Reply:
There are several ways to use the domain command, depending on the specific needs of your project:

1. ```bccli domain generate (domainName)``` (This one is the default): This generates the necessary files for the domain, handler, and usecase. For the repository, it only creates the folder without generating any files.

2. ```bccli domain generate (domainName) -d (database)```: This generates the files for the domain, handler, usecase, and additionally, a complete repository layer tailored to the specified database.

3. ```bccli domain generate (domainName) <Flags>```: This allows for more granular control. Based on the flags provided, it will generate files for specific components only. For example, using the --handler flag would generate the handler file without creating other components.

Reply:
So it defaults to generate all at once correct? what about changing the phrase from "You can choose to create specific components or generate all at once." to "By default it generate all component at once, but you can choose one or multiple."